shell/src/main/java/tachyon/shell/command/ChgrpCommand.java

```diff
@@ -50,19 +54,36 @@ protected int getNumOfArgs() {
   }
 
   @Override
-  public void run(String... args) throws IOException {
+  protected Options getOptions() {
```

Comment:
I'm planning to refactor AbstractAclCommand and move the cmd method to it's own cmd class, since chgrpr classes are no longer needed.